ddspipe_core/include/ddspipe_core/core/DdsPipe.hpp

```diff
@@ -212,6 +212,14 @@ class DdsPipe
     void updated_endpoint_nts_(
             const types::Endpoint& endpoint) noexcept;
 
+    /**
+     * @brief Method called every time a new endpoint has been discovered, removed, or updated.
```

Comment:
Missing `@return` value in the description. Also in `is_endpoint_relevant_` method.